static/styles/index.css

```diff
@@ -0,0 +1,11309 @@
+@charset "UTF-8";
+/*!
+* Start Bootstrap - Landing Page v6.0.6 (https://startbootstrap.com/theme/landing-page)

```

Comment:
We are importing bootstrap already by an script, why are you adding it in this index file too?